tests/plugins/dialog/plugin.js

```diff
@@ -401,6 +435,16 @@
 
 			editor.execCommand( 'testDialog6' );
 			wait();
+		},
+
+		// (#2277)
+		'test default values': function() {
+			this.editorBot.dialog( 'testDialog1', function( dialog ) {
+				assert.areEqual( 'text1', dialog.getContentElement( 'info', 'text1' ).getValue(), 'text1 field has invalid value' );
+				assert.areEqual( 'text2', dialog.getContentElement( 'info', 'text2' ).getValue(), 'text2 field has invalid value' );
+
+				dialog.getButton( 'cancel' ).click();
```

Comment:
There are several issues for this way of closing a dialog:

1. It is called only if no assertions fails (meaning that if something fails, rest of the cases is messed).
1. It requires copy and pasting this code to each assertion.

You could do it better, by putting this kind of cleanup code into `tearDown` funciton of test suite. Call `CKEDITOR.dialog.getCurrent()` and if there is any then close it.

I have added one code where I intentionally copied the code from here, so you can do the extraction explained above.